CHANGELOG.rst

```diff
@@ -2,6 +2,42 @@
 CHANGELOG
 =========
 
+2.9.0
+=====
+
+**ENHANCEMENTS**
+
+* Added support for heterogeneous instance types with Slurm Scheduler.
+* Added CLI utility to convert ParallelCluster configuration files from single to Heterogeneous instance types.
+* Extended DCV support to ARM instances.
+* Added support for Torque-Slurm wrapper.
+* Extended support for disabling hyperthreading to instances (like \*.metal) that don't support CpuOptions in launch
+  template.
+* Enable nfsv4 for computing instances file system mounting.
+
+**CHANGES**
+
+* Introduced new configuration sections and parameters to support Heterogeneous Instance Types.
+* Optimized scaling logic with Slurm scheduler, no longer based on Auto Scaling Groups.
+* A Route53 private hosted zone is now created together with the cluster and used in DNS resolution inside cluster
+  nodes.
+* Upgrade Slurm to version 20.02.4.
+* Upgrade NICE DCV to version 2020.1-9012.
+* Use private ip instead of master node's hostname when mounting shared NFS drives.
+* Added new log streams to CloudWatch: chef-client, clustermgtd, computemgtd, slurm_resume, slurm_suspend.
+* Removed dependency on cfn-init in compute nodes bootstrap.
+* Added support for queue names in pre/post install scripts.
+* Use PAY_PER_REQUEST billing mode for DynamoDb table in govcloud regions.
+
+**BUG FIXES**
+
+* Solved dpkg lock issue with Ubuntu that prevented custom AMI creation in some cases.
+* Added/improved sanity checks for some configuration parameters.
+* Prevented ignored changes from being reported in ``pcluster update``'s output.
+* Fixed incompatibility issues with python 2.7 for ``pcluster update``.
+* Handled AMI validation error when AMI not found.
```

Comment:
?

Reply:
https://github.com/aws/aws-parallelcluster/commit/b339bcd52da2472883675e3700c369783071b93d

Maybe can be considered part of "Add/improve sanity checks for some configuration parameters."